lib/fog/compute/proxmox/models/servers.rb

```diff
@@ -56,8 +56,10 @@ def id_valid?(vmid)
         def get(vmid)
           requires :node
           path_params = { node: node, type: type, vmid: vmid }
-          server_data = service.get_server_status path_params
+          server_data = service.get_server_status(path_params)
+          server_data = server_data.first if server_data.is_a?(Array)
           config_data = service.get_server_config path_params
+          config_data = config_data.first if config_data.is_a?(Array)
```

Comment:
Is there a reason why the server_data / config_data is maybe a Array? Would it make sense, to pick another item of the array items instead only the fist one?